docs/_renderer.py

```diff
@@ -142,9 +144,37 @@ def render_annotation(self, el: exp.ExprName):
         return f"[{el.name}](`{el.canonical_path}`)"
 
     @dispatch
-    def summarize(self, el: dc.Object | dc.Alias):
-        result = super().summarize(el)
-        return html.escape(result)
+    # Overload of `quartodoc.renderers.md_renderer` to fix bug where the descriptions
+    # are cut off and never display other places. Fixing by always displaying the
+    # documentation.
+    def summarize(self, obj: Union[dc.Object, dc.Alias]) -> str:
+        # get high-level description
+        doc = obj.docstring
+        if doc is None:
+            docstring_parts = []
+        else:
+            docstring_parts = doc.parsed
+
+        if len(docstring_parts) and isinstance(
+            docstring_parts[0], ds.DocstringSectionText
+        ):
+            description = docstring_parts[0].value
+
+            # ## Approach: Always return the full description!
+            return description
```

Comment:
This will now always display the full description rather than being randomly cut off